### ISSUE TITLE
Give option to manually trigger scheduled Triggers

### ISSUE THREAD

**Issue by waj** [instedd/mbuilder#214]
Originally reported by @edjez in BitBucket

Give me an option to "Run Now" a scheduled Trigger (or any trigger, prompting for appropriate arguments.. )
Empowers users to run manual tasks

**Comment by nditada** [comment]
@jkicillof can you provide a design for this?

**Comment by jkicillof** [comment]
The "Run Now" button only make sense on schedulled triggers, common and validation triggers can be triggered on the testing sandbox. 
![image](https://cloud.githubusercontent.com/assets/5031636/17345341/e391b310-58db-11e6-9ee7-ce0b4e9aca7c.png)

**Comment by asterite** [comment]
Can the "Run Now" button be in the list page? Because to run a trigger we first need to save it. Otherwise it can't be confusing as to what happens when you are editing a trigger, click "Run now!", but don't save the trigger (where is it saved?)

I also don't know how to implement it otherwise...

**Comment by asterite** [comment]
And by the way, we also have this problem when editing validation triggers for a table field: unless the trigger is saved beforehand, trying to edit a validation trigger will crash the app

**Comment by nditada** [comment]
@jkicillof: Yes, 👍  on the having the button on the list page, not inside the detail. Otherwise user have to open the trigger to be able to run it.
